index.bs

```diff
@@ -1402,6 +1405,87 @@ lt="ReadableStreamDefaultReader(stream)">new ReadableStreamDefaultReader(<var>st
   1. Perform ! ReadableStreamReaderGenericRelease(*this*).
 </emu-alg>
 
+<h5 id="default-reader-iterator" method for="ReadableStreamDefaultReader">
+  iterator({ cancel = true } = {})
```

Comment:
There is no precedent for an `iterator` method anywhere in the platform and I'm reluctant to start that now. Just having `@@asyncIterator` is good enough IMO.

Reply:
`iterator()` came from the linked issue, i'm unsure what to do here.

Reply:
Yeah we should have this method. But we should flip it to preventCancel = true. https://w3ctag.github.io/design-principles/#prefer-dict-to-bool third paragraph.

Reply:
How about "getIterator" or "iterate"? The first provides consistency with the existing "getReader". I do prefer a name with a verb in it.

Reply:
@domenic The default for this would be `preventCancel = false`, right?

Reply:
Hmm, now I'm thinking "values()" for symmetry with sync iterators?

Reply:
OTOH that has the potential to be confused _as_ a sync iterator…